apps/src/applab/PhoneFrame.jsx

```diff
@@ -0,0 +1,69 @@
+import Radium from 'radium';
+import commonStyles from '../commonStyles';
+import color from '../color';
+import applabConstants from './constants';
+import experiments from '../experiments';
+
+const RADIUS = 30;
+
+const styles = {
+  phoneFrame: {
+    display: 'block',
+    height: 50,
+    backgroundColor: color.lighter_gray,
+  },
+  phoneFrameRunning: {
+    backgroundColor: color.charcoal
+  },
+  phoneFrameTop: {
+    borderTopLeftRadius: RADIUS,
+    borderTopRightRadius: RADIUS,
+  },
+  phoneFrameBottom: {
+    borderBottomLeftRadius: RADIUS,
+    borderBottomRightRadius: RADIUS,
+  },
+  nonResponsive: {
+    maxWidth: applabConstants.APP_WIDTH,
+  }
+};
+
+
+const PhoneFrame = React.createClass({
+  propTypes: {
+    showFrame: React.PropTypes.bool.isRequired,
+    isRunning: React.PropTypes.bool.isRequired
+  },
+
+  render: function () {
```

Comment:
I would think what you have there would work, though it feels a little strange to export it, then change it, i.e. I might prefer the following:
```
function PhoneFrame(props) {
  return ...;
}
PhoneFrame.propTypes = {
  // ...
};
export default PhoneFrame
```

I'm wondering if we're perhaps showing too much eagerness to use stateless functional components. I think they're great for something that's just a couple lines long and only uses internally (i.e. not exported), but I'm starting to question if they have as much value for anything bigger than that.